src/logic/operate.js

```diff
@@ -0,0 +1,35 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const numOne = new Big(numberOne);
+  const numTwo = new Big(numberTwo);
+  const minusOne = new Big(-1);
+  let result;
+
+  switch (operation) {
+    case '+':
+      result = numOne.plus(numTwo);
+      break;
+    case '-':
+      result = numOne.minus(numTwo);
+      break;
+    case 'X':
+      result = numOne.times(numTwo);
+      break;
+    case '÷':
+      result = numOne.div(numTwo);
+      break;
+    case '%':
+      result = numOne.mod(numTwo);
```

Comment:
- [ ] `%` This is a percentage operator, not a modulus operator. So instead divide the number by 100.